.github/workflows/ci.yml

```diff
@@ -18,31 +18,41 @@ jobs:
       matrix:
         scala-version: [ "2.12", "2.13", "3" ]
         target-platform: [ "JVM", "JS", "Native" ]
+        java: [ "11", "21" ]
         exclude:
           - scala-version: "2.12"
             target-platform: "Native"
+          - scala-version: "2.12"
+            java: "21"
           - scala-version: "2.13"
             target-platform: "Native"
+          - java: "21"
+            target-platform: "Native"
+          - java: "21"
+            target-platform: "JS"
```

Comment:
so ... we are trying to only include specific axes? maybe we can either (1) use include instead of exlucde, if possible; (2) if not, at least add a comment, what are the desired combinations that should be included

Reply:
Thanks for the hint, somehow I wasn't aware that there's `include`. Turns out it can be combined with `exclude`, because exclusions are resolved first, so our rules will become:
```yaml
        exclude:
          - java: "21"
          - scala-version: "2.12"
            target-platform: "Native"
          - scala-version: "2.13"
            target-platform: "Native"
        include:
          - java: "21"
            scala-version: "2.13"
            target-platform: "JVM"
          - java: "21"
            scala-version: "3"
            target-platform: "JVM"
```

Looks clearer to me.